### PR TITLE
Multi-Delegate

### PR DESCRIPTION
**Pull request by remmiculous**
**Added:**

- Multi-Delegate button in Stake Page
- Drop-down menu with check-boxes to select delegates
- Select all button in Drop-down menu
- Modified Success to better suit multiple delegation.

### CODE REVIEW

**Review by poojachigitey — APPROVED** [review]

File: src/containers/Stake/DelegateDialog/MultiValidatorSelectField.js [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+                    <Divider className={'divider'} />
+                    {validatorList && validatorList.length > 0 &&
+                        validatorList.map((item, index) => {
+                            const image = item && item.description && item.description.identity &&
+                                    props.validatorImages && props.validatorImages.length &&
+                                    props.validatorImages.filter((value) => value._id === item.description.identity.toString());
+
+                            return (
+                                <MenuItem
+                                    key={item.key || item.value || item.name || item.type ||
```
Comment (SrikanthSoparla) [inline src/containers/Stake/DelegateDialog/MultiValidatorSelectField.js]: use index here

File: src/containers/Stake/DelegateDialog/MultiValidatorSelectField.js [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+                                            item.operator_address}
+                                    className={'mv_menuItem'}
+                                    value={item.value || item.name || item.type ||
```
Comment (SrikanthSoparla) [inline src/containers/Stake/DelegateDialog/MultiValidatorSelectField.js]: send only address here

File: src/containers/Stake/DelegateDialog/MultiValidatorSelectField.js [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+                                            src={image[0].them[0].pictures.primary.url}/>
+                                        : item.description && item.description.moniker
+                                            ? <span
+                                                className="image"
+                                                style={{ background: colors[index % 6] }}>
+                                                {item.description.moniker[0]}
+                                            </span>
+                                            : <span className="image" style={{ background: colors[index % 6] }}/>}
+                                    {item.name ? item.name : item.type
+                                        ? item.name : item.description && item.description.moniker}
```
Comment (SrikanthSoparla) [inline src/containers/Stake/DelegateDialog/MultiValidatorSelectField.js:93]: {item.name
     ? item.name
     : item.type
            ? item.type : item.description && item.description.moniker}

File: src/containers/Stake/DelegateDialog/index.css [hunk 4]
```diff
@@ -168,3 +168,77 @@
...
+.mv_select {
+    border: solid #696969 0.3px;
+    height: 48px;
+}
+
+.mv_select .MuiOutlinedInput-notchedOutline {
+    border: none !important;
+}
+
+.mv_select em{
```
Comment (SrikanthSoparla) [inline src/containers/Stake/DelegateDialog/index.css]: add space between em{

**Review by SrikanthSoparla — APPROVED** [review]

**Pull request merged**